MIDIchlorians/MIDIchlorians/Session.swift

```diff
@@ -52,6 +52,10 @@ class Session: Object {
         return sessionName
     }
 
+    func getPad(page: Int, indexPath: IndexPath) -> Pad {
```

Comment:
Cool, I'm gonna merge this in first, when you merge yours you can prob keep both methods, and we will deprecate this one in a future PR